waku/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -35,15 +35,16 @@ proc createTableQuery(): string =
   " version INTEGER NOT NULL," &
   " timestamp BIGINT NOT NULL," &
   " id VARCHAR NOT NULL," &
+  " messageHash VARCHAR NOT NULL," &
   " storedAt BIGINT NOT NULL," &
-  " CONSTRAINT messageIndex PRIMARY KEY (storedAt, id, pubsubTopic)" &
+  " CONSTRAINT messageIndex PRIMARY KEY (storedAt, messageHash)" &
```

Comment:
First half of your question:
short answer: `storedAt` is required otherwise, the integrity of the DB will be broken.
long answer: https://github.com/waku-org/nwaku/blob/master/tests/testlib/wakucore.nim#L54 Payload and meta are optional here, if we remove `storedAt` from primary key, then some test cases will fail. i.e. any app sending the data might keep the same data (payload, meta, content and pubsub topic, all those required for `messageHash` computation)

Reply:
I agree with your statement @jm-clius. Better have the `messageHash` as the primary ID.

Reply:
in our test cases at many places the `timestamp` is used as same along with other things, that will make the DB integrity break if we only use `messageHash` as the primary key. So either we:
1. change all the test cases where the time is set as same, or
2. we don't change primary key for now
WDYT @jm-clius @Ivansete-status